about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sprintplex Summer Coding Camp | Giving the youth a chance</title>
+    <link rel="shortcut icon" type="image/jpg" href="./build/images/favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./build/css/style.css" />
+    <link rel="stylesheet" href="./build/css/main.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__user-bar">
+        <ul class="header__user row">
+          <li><i class="fa-brands fa-facebook-f"></i></li>
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><p class="header__language">English</p></li>
+          <li><p class="header__user-page">My page</p></li>
+          <li><p class="header__logout">Logout</p></li> 
+        </ul>
+      </div>
+      <nav class="header__navbar row">
+        <a href="index.html" class="header__home-link">
+          <img
+            src="./build/images/sprintplex-c-c.png"
+            alt="Sprintplex coding camp logo"
+            class="header__brand-img"
+          />
+        </a>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="about.html" class="header__nav-link">About</a>
```

Comment:
Ask requested by the previous reviewer kindly make sure it's possible to navigate between the pages, there's a link from the Home page to the About page and vice versa. I can see that all the menu links of the mobile version goes to the about page including the logo at the bottom. Kindly consider changing maybe the links of the `Program` and logo to direct to the main page :pray: